src/Faker/Extension/CountryExtension.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Faker\Extension;
+
+/**
+ * @experimental This interface is experimental and does not fall under our BC promise
+ */
+interface CountryExtension extends Extension
```

Comment:
Unrelated to this PR but maybe we should add alpha2 and alpha3 numbers too from the Iso spec?

This way we can adapt the country extension in core as well and localized extensions can just extend from there.

What is your view on that?

Reply:
Maybe, I have no strong opinions for or against. I'll try to focus on the rewrite atm and not adding new features.

Reply:
Agree, lets add them later